definitions/npm/zxcvbn_v4.x.x/flow_v0.25.x-/zxcvbn_v4.x.x.js

```diff
@@ -0,0 +1,204 @@
+declare module 'zxcvbn' {
+  declare export type Result = {
```

Comment:
This could work better as exact and immutable type, see [prefer immutability](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-immutability) and [prefer exactness](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-exactness)